L1Trigger/L1TCalorimeter/plugins/L1TCaloRCTToUpgradeConverter.cc

```diff
@@ -100,17 +100,16 @@ L1TCaloRCTToUpgradeConverter::produce(edm::Event& iEvent, const edm::EventSetup&
     ROOT::Math::LorentzVector<ROOT::Math::PxPyPzE4D<double> > p4(0,0,0,0);
 
 
-    // create new format
-    //CaloRegion region;
-    CaloRegion region(*&p4,
-			   0.,
-			   0.,
-			   (int) rgn->et(),
-			   (int) rgn->id().ieta(),
-			   (int) rgn->id().iphi(),
-			   hwQual,
-			   0,
-			   0);
+    // create new format  TODO: not filled with necessary information
```

Comment:
Ok, I see now that in the [float implementation](https://github.com/cms-l1t-offline/cmssw/blob/EtSumAlg-devel/L1Trigger/L1TCalorimeter/src/firmware/Stage1Layer2EtSumAlgorithmImpPP.cc#L147) we use dijet phi.
I don't have any information on this part of the firmware, so as of now I cannot add it to the emulation.  I'll add a skeleton to be filled later.